idp_mgmt_docs.adoc

```diff
@@ -9,11 +14,3 @@ The identity configuration management for Kubernetes is a Technology Preview fea
 
 The identity configuration management for Kubernetes is a software operator that enhances cluster fleet management. It enables you to quickly and easily propagate OAuth authentication configuration across your fleet of OpenShift Container Platform clusters regardless of cloud provider or data center.
 
-See the following documentation:
```

Comment:
@itdove Can I assume this entry is going to appear still somehow in the main page of the generated html pages?

Reply:
yes here the upstream docs
https://identitatem.github.io/idp-mgmt-docs/
https://identitatem.github.io/idp-mgmt-docs-upstream/ebook.pdf